src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -259,9 +262,10 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
           this.copySingleSession(requestList[0]);
         }
         if (requestList.length > 1) {
-          forkJoin(requestList).subscribe(() => {
-            this.showCopyStatusMessage();
-          });
+          forkJoin(requestList).pipe(finalize(() => this.sessionEditFormModel.isCopying = false))
```

Comment:
Line break like line 451 too?

Reply:
Here is the link to the requested video:
https://drive.google.com/file/d/1P6JeG2-OiGmEqb6_4hxVWj0tAVccttlW/view?usp=sharing

As soon as I press the save button it disappears and the edit button takes its place. The spinner doesn't show because the save button is removed before it appears. To fix it I changed the logic in the HTML file to allow the save button to stay until the save process finishes. (See after snapshots for proof: https://drive.google.com/file/d/1K_8a5zzV4VkIbe5xbap_vCa7EmGOwwu1/view?usp=sharing)

Reply:
I have added the line break.